### ISSUE TITLE
Dependencies are old

### ISSUE THREAD

**Issue by dolmen** [cernops/golbd#10]
From go.mod:
```
require (
	github.com/miekg/dns v0.0.0-20160605072344-799de7044d95
	github.com/reguero/go-snmplib v0.0.0-20181019092238-e566f5619b55
)
```

The dependencies have not been updated to recent versions. Is it on purpose?

**Comment by dolmen** [comment]
One explanation about not using latest version of `github.com/miekg/dns` is given here: https://github.com/cernops/golbd/pull/5#issuecomment-1097917831
> As matter of fact, we cannot use the latest version of github.com/miekg/dns because we need to support TSIG keys using HmacMD5 until the software supporting the DNS servers at CERN is updated and github.com/miekg/dns has dropped support for it in later versions.

**Comment by reguero** [comment]
Indeed we are waiting for github.com/miekg/dns.
For github.com/reguero/go-snmplib we need some mods/fixes to go-snmplib that I had done but they have been merged upstream a while ago so I guess that we could point there...

**Comment by dolmen** [comment]
Dependency on `github.com/mielg/dns` has been upgraded by d891dacb97c6e1c788d4790b9a4f22261d28a392. Was it wanted?

**Comment by reguero** [comment]
Indeed, I understand that v1.0.0 was required for DNS Server support but is is still older than commit  93945c28 - Remove HMAC-MD5 support from TSIG (#1187) (Sat Oct 24 13:57:51 2020) <Tom Thorogood>
so it is OK for us.
